subsys/bluetooth/mesh/access.h

```diff
@@ -16,9 +16,11 @@ void bt_mesh_elem_register(struct bt_mesh_elem *elem, uint8_t count);
 
 uint8_t bt_mesh_elem_count(void);
 
-/* Find local element based on unicast or group address */
+/* Find local element based on unicast address */
```

Comment:
Why del `or group` ?
I noticed that `bt_mesh_elem_find` function has not changed?

Reply:
bt_mesh_elem_find has been changed but the delta makes it difficult to see because a part of it has been moved down into a new function. The for loop in bt_mesh_has_addr which searches through group addresses of local elements was originally in bt_mesh_elem_find.